src/main/resources/application.conf

```diff
@@ -257,7 +257,7 @@ grothendieck {
     sync-retry-interval = 5.seconds
 
     # Response time-out from peer during sync. If a peer fails to respond within this limit, it will be blacklisted
-    peer-response-timeout = 10.seconds
+    peer-response-timeout = 3.minutes
```

Comment:
3 minutes is a lot. I doubt someone will respond in more than 1 minute. Maybe 1 then?